<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>89</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>发送文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>89</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>获取文件列表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>89</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>运行</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>89</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>暂停运行</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>89</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>编辑模式</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>509</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="ipLabel">
       <property name="text">
        <string>IP 地址：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ipLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="portLabel">
       <property name="text">
        <string>端口</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="portLineEdit"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="connButton">
       <property name="text">
        <string>连接</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="disconnButton">
       <property name="text">
        <string>断开</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>501</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="tickButton">
       <property name="text">
        <string>倒计时</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sketchButton">
       <property name="text">
        <string>剪影</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="schButton_1">
       <property name="text">
        <string>预定模式一</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="schButton_2">
       <property name="text">
        <string>预定模式二</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="schButton_3">
       <property name="text">
        <string>预定模式三</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>526</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOperation">
    <property name="title">
     <string>Operation</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisConnect"/>
    <addaction name="actionSend_File"/>
   </widget>
   <addaction name="menuOperation"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </action>
  <action name="actionDisConnect">
   <property name="text">
    <string>&amp;DisConnect</string>
   </property>
  </action>
  <action name="actionSend_File">
   <property name="text">
    <string>&amp;Send File</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>connButton</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>connectServer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playButton</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>setPlay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pauseButton</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>setPause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>461</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>disconnButton</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>disconnServer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editButton</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>editSchema()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tickButton</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>sendTimeTick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sketchButton</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>setSketch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>schButton_1</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>sendSchema_1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>schButton_2</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>sendSchema_2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>schButton_3</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>sendSchema_3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sendButton</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>setSendFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>connectServer()</slot>
  <slot>disconnServer()</slot>
  <slot>setPlay()</slot>
  <slot>setPause()</slot>
  <slot>setSendFile()</slot>
  <slot>editSchema()</slot>
  <slot>sendTimeTick()</slot>
  <slot>sendSchema_1()</slot>
  <slot>sendSchema_2()</slot>
  <slot>sendSchema_3()</slot>
  <slot>setSketch()</slot>
 </slots>
</ui>
